services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -1290,6 +1258,45 @@ public IOFSwitch lookupSwitch(DatapathId dpId) throws SwitchNotFoundException {
                 .collect(Collectors.toList());
     }
 
+    @VisibleForTesting
+    OFInstructionMeter installMeterAndBuildInstruction(IOFSwitch sw, long meterId, long packetsLimit,
```

Comment:
You should mention "default" rules in method name. And `AndBuildInstruction` is extra/useless info, because or return value type.